virt/monitoring/virt-runbooks.adoc

```diff
@@ -219,6 +219,11 @@ Runbooks for the {VirtProductName} Operator are maintained in the link:https://g
 
 * link:https://github.com/openshift/runbooks/blob/master/alerts/openshift-virtualization-operator/SSPTemplateValidatorDown.md[View the runbook] for the `SSPTemplateValidatorDown` alert.
 
+[id="virt-runbook-SSPOperatorDown"]
```

Comment:
```suggestion
[id="virt-runbook-ssp-operatordown_{context}"]
```
Needs context tag, and likely, lowercase.
See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-titles-and-section-headings. Although not listed there, if you scroll up, all of the guidance around IDs for assemblies and modules stipulate lowercase in IDs.

Reply:
@ShaunaDiaz I followed the ID pattern established in the module - https://raw.githubusercontent.com/openshift/openshift-docs/main/virt/monitoring/virt-runbooks.adoc

Still, you have a point that it doesn't adhere to the naming conventions, I'll fix it in another commit.